app/assets/javascripts/primer.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
I wonder whether or not we need to commit this file?

It will be needed for publishing the npm package and the gem, but as it’s an artefact generated from other source, maybe we should not check it in?

Reply:
I think we need to commit it, otherwise it won't be available when vendoring the gem